components/lib/recipes.js

```diff
@@ -0,0 +1,110 @@
+class RecipeClass {
+  constructor(id, title, subtitle, picture) {
+    if (!id || !title || !subtitle) {
+      throw new Error("Id, title, subtitle and workingTime must be provided");
+    }
+    this.id = id;
+    this.title = title;
+    this.subtitle = subtitle;
+    this.picture = picture || "recipe-placeholder.png";
+  }
+}
+
+const recipes = [
+  {
+    id: 1,
+    title: "Spaghetti Bolognese",
+    subtitle: "with ground beef",
+    picture: "Bolognese.png",
+    ingredients: [
+      "500g spaghetti",
+      "400g ground beef",
+      "1 onion",
+      "2 garlic cloves",
+      "400g canned tomatoes",
+      "1 tbsp tomato paste",
+      "1 tsp dried oregano",
+      "1 tsp dried basil",
+      "Salt and pepper",
+      "Olive oil",
+    ],
+    instructions:
+      "1. Finely chop the onion and garlic, then sauté them in olive oil. Add the ground beef and cook until browned. \n2. Stir in the tomato paste, canned tomatoes, oregano, basil, salt, and pepper. Simmer for 20-30 minutes. \n3. Meanwhile, cook the spaghetti according to the package instructions. \n4. Serve the Bolognese sauce over the drained spaghetti and enjoy.",
+  },
+  {
+    id: 2,
+    picture: "caesar-salad.png",
```

Comment:
You might want to use the same order as in id: 1 even though it is not relevant for an object. But maybe it is helpful for yourself